linkerd/config/src/main/scala/io/buoyant/linkerd/config/types/DtabDeserializer.scala

```diff
@@ -11,3 +11,11 @@ class DtabDeserializer extends ConfigDeserializer[Dtab] {
       Dtab.read(_parseString(parser, ctx))
     }
 }
+
+class DtabSerializer extends ConfigSerializer[Dtab] {
```

Comment:
I don't think you can discover inner classes via ServiceLoader :cry: 

Unless someone starts having :muscle: feelings, I'm going leave this alone.